src/platform/packages/shared/response-ops/rule_form/src/utils/get_initial_multi_consumer.ts

```diff
@@ -53,7 +55,7 @@ export const getInitialMultiConsumer = ({
   }
 
   // If o11y is in the valid consumers, just use that
-  if (validConsumers.includes(AlertConsumers.OBSERVABILITY)) {
+  if (isServerless && validConsumers.includes(AlertConsumers.OBSERVABILITY)) {
```

Comment:
Why this change?

Reply:
This is because `validConsumers` of o11y solution always has `AlertConsumers.OBSERVABILITY`, however we don't want it for some rules like (custom threshold, esql etc.) which have other consumers like logs, metrics etc. 
However if it is serverless, we want all rules to have `AlertConsumers.OBSERVABILITY` consumer.

Reply:
We need tests for this file 😬

Reply:
You are right 😅 , I will add some.

Reply:
Maybe this line deserves a comment?

Reply:
done https://github.com/elastic/kibana/pull/206774/commits/4cea1c1909801cca0f0d65ab36c6236340dd2982